api/src/main/java/org/openmrs/api/impl/OrderServiceImpl.java

```diff
@@ -315,7 +315,7 @@ private boolean areDrugOrdersOfSameOrderableAndOverlappingSchedule(Order firstOr
 		        && !OpenmrsUtil.nullSafeEquals(firstOrder.getPreviousOrder(), secondOrder)
 		        && OrderUtil.checkScheduleOverlap(firstOrder, secondOrder)
 		        && firstOrder.getOrderType().equals(
-		            Context.getOrderService().getOrderTypeByUuid(OrderType.DRUG_ORDER_TYPE_UUID));
+		            Context.getOrderService().getOrderTypeByName(OpenmrsConstants.DRUG_ORDER_TYPE_NAME));
```

Comment:
I don't understand why we are making this change.  We still want to look up by these uuids - and if they are found, continue as before.  But if not found, we should try to find a unique match by class.  So if orderType is null by looking up by DRUG_ORDER_TYPE_UUID, then use the new methods you have added to look up OrderType based on "org.openmrs.DrugOrder" class, and if only one is found, then return it.  The "name" of the order type is not something we should be looking up.